LQResultProducer.py

```diff
@@ -4610,6 +4607,7 @@ def GetMuMuScaleFactors( selection, FileDirectory, controlregion_1, controlregio
 	# print QuickIntegral(t_ZJets,selection + '*' + controlregion_1,1.0)
 	# sys.exit()
 	selection_data = selection.split('*(fact')[0]
+	selection_data = selection.split('*scaleWeight')[0]
```

Comment:
Are you sure that the scaleWeight is always the last element in the string? If not, you will lose anything after it. Maybe we should in the future just set scaleWeight to 1 for data? It thought you were already doing that actually...

Reply:
Yeah, the only other place scaleWeights are applied to selection is in GetMuMuScaleFactorsMod and GetMuNuScaleFactorsMod both of which apply the weights to the very end of the selection. And yes, I agree the scaleWeights should be set to 1 in the analyzer... something for the next round of trees.